### DISCUSSION TITLE
Didn't actually download anything

### DISCUSSION THREAD

**Discussion by Hawkbone** [shaked6540/YoutubePlaylistDownloader#186] [Q&A]
Tried downloading my music playlist but it didn't work. The queue says that its completed but the folder that the files were supposed to go to is completely empty, I made sure to edit the folders options to show hidden and compressed files but that didn't do anything. I tried downloading it a second time and it created a new folder within the original destination folder that also had nothing in it.

![Screenshot 2023-08-24 114950](https://github.com/shaked6540/YoutubePlaylistDownloader/assets/46417811/74f62bd8-70c5-4742-846f-b43f7af7a96b)
![Screenshot 2023-08-24 115114](https://github.com/shaked6540/YoutubePlaylistDownloader/assets/46417811/016b2f87-c69f-47f4-877c-d1deb5fd35aa)

**Comment by shaked6540** [comment]
If the files don't show up after you download a playlist, try running the program as administrator or change the save directory to a different drive! (C drive is usually protected by windows, save it under D or any other drive)

**Reply by jascharl** [reply]
Rnning the program as Admin worked for me. Thank you.
